deno.jsonc

```diff
@@ -2,14 +2,13 @@
     "lock": false,
     "tasks": {
         "check": "deno cache --check=all src/mod.ts",
-        "backport": "deno run --no-prompt --allow-read=. --allow-write=. https://deno.land/x/deno2node@v1.9.0/src/cli.ts tsconfig.json",
+        "contribs": "npx --yes all-contributors-cli@6",
+        "backport": "deno run --no-prompt --allow-read=. --allow-write=out/ bundling/build-cjs.ts src/mod.ts",
         "test": "deno test --seed=123456 --parallel ./test/",
         "dev": "deno fmt && deno lint && deno task test && deno task check",
         "coverage": "rm -rf ./test/cov_profile && deno task test --coverage=./test/cov_profile && deno coverage --lcov --output=./coverage.lcov ./test/cov_profile",
         "report": "genhtml ./coverage.lcov --output-directory ./test/coverage/ && echo 'Point your browser to test/coverage/index.html to see the test coverage report.'",
-        "bundle": "cd bundling && ./bundle-all.sh",
-        "bundle-web": "mkdir -p out deno_cache && cd bundling && DENO_DIR=$PWD/../deno_cache deno run --unstable --quiet --allow-net --allow-read --allow-env=DENO_DIR,XDG_CACHE_HOME,HOME,DENO_AUTH_TOKENS --allow-write=../out,$PWD/../deno_cache bundle-web.ts dev ../src/mod.ts",
-        "contribs": "deno run --allow-env --allow-read --allow-write=. --allow-net=api.github.com npm:all-contributors-cli"
```

Comment:
Accidentally removed this while resolving conflicts 🙈 pin to a major release and tighten permission when restoring.